docs/source/support.rst

```diff
@@ -0,0 +1,39 @@
+=================
+ Support options
+=================
+
+How to get additional help
+==========================
+
+Free support
+============
```

Comment:
Shouldn't these be a different underline to make it a subheader of "How to get additional help"?